web/docs/understand/query-language/expressions.md

```diff
@@ -149,18 +149,16 @@ types because an alias is a strict refinement of an existing type.
 #### Meta Extractor
 
 Meta extractors have the forms `#E` where `E` is from the fixed set of tokens
-`type`, `field`, and `import_time`. They work on the event metadata (e.g., their
+`type`,  and `import_time`. They work on the event metadata (e.g., their
```

Comment:
```suggestion
type and import timestamp. They work on the event metadata (e.g., their
```

Reply:
That doesn't make sense in the sentence. I' rather shorten it to this:

> Meta extractors have the form `#extractor`.

And remove the whole second half of the sentence. It doesn't add anything with the list below.